node/node/src/inherent_data.rs

```diff
@@ -117,6 +125,27 @@ where
 		)
 		.await?;
 
+		#[cfg(test)]
+		let payouts = BlockParticipationInherentDataProvider::new_cardano_stake_based(
+			client.as_ref(),
+			stake_distribution_data_source.as_ref(),
+			parent_hash,
+			*slot,
+			mc_epoch_config,
+			config.sc_slot_config.slot_duration,
+		)
+		.await?;
+		#[cfg(not(test))]
+		let payouts = BlockParticipationInherentDataProvider::new_cardano_stake_based_if_pallet_present(
+			client.as_ref(),
+			stake_distribution_data_source.as_ref(),
+			parent_hash,
+			*slot,
+			mc_epoch_config,
+			config.sc_slot_config.slot_duration,
+		)
+		.await?;
```

Comment:
I am not sure why we are losing comments history: https://github.com/input-output-hk/partner-chains/pull/515#discussion_r1976436874

Please unify this calls and remove macro